### DISCUSSION TITLE
Issues with post processing command

### DISCUSSION THREAD

**Discussion by deviantan** [neperfepx/FEPX#120] [Q&A] [answered]
I am unable to run the post processing command - "fepx-pp simple_simulation" I am getting an error "fepx-pp: command not found". Please let me know how I can solve this issue.
I have installed fepx properly and the simulation is also running but the fepx-pp command is unrecognized. Any help would be greatly appreciated. Thanks

**Comment by rquey** [comment, marked as answer]
This command is mentioned in the "Simple Simulation" tutorial, but you actually don't need to run it at all.

We'll fix the tutorial in the next release.
